docker/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "2"
+services: 
+  rocketchat-tests-based-on-splinter:
+    image: cusdeb/rocketchat-tests-based-on-splinter
+    command: "-s all"
+    links:
+      - "rocketchat:rocketchat"
+    environment: 
+      - USERNAME=admin
+      - PASSWORD=qwerty12+
+      - ADDR=rocketchat
+      - PORT=8006
+      - HOST=http://rocketchat:8006
+      - BOT_NAME=meeseeks
+      - BRANCH=wizard-integration
```

Comment:
* the branch doesn't exist any more
* use `${BRANCH}` here to allow changing the branch via `.env`